### PR TITLE
Added tab structure and map to search page, integrated zipcodeInput compent

### LINKED ISSUE #11: Results tabs
- [ ] Implement the layout for the map and list tabs inside the result container

### PR DESCRIPTION
**Pull request by ivinjose**
Fixes #11 

Changes included --
- [x] Added tab layout to search page
- [x] Integrated maps to search page
- [x] Integrated zipcodeInput component to search page header

### CODE REVIEW

**Comment by shidhincr** [comment]
:+1:  I'll review this once I get time

File: app/scripts/ui/ZipcodeDialog.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var ZipcodeDialog = React.createClass({
         return(
             <Modal title="Enter your zipcode" closeButton={false} onRequestHide={this.onHide}>
                 <div className="modal-body">
-                    <ZipcodeInput />
+                    <ZipcodeInput ackWhenValid={true} />
```
Comment (shidhincr) [inline app/scripts/ui/ZipcodeDialog.js]: What is ackWhenValid ?

Comment (ivinjose) [inline app/scripts/ui/ZipcodeDialog.js]: Its the prop using which you can show/hide the 'Valid zipcode' message shown by zipcodeInputComponent. 

In the search page header, when we show zipcodeInput field, this message looks bad and not really required. So I added this prop as flag.

File: app/scripts/ui/results.js [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+var TabPane = require('react-bootstrap/TabPane');
+var TabbedArea = require('react-bootstrap/TabbedArea');
 
 var Results = React.createClass({
     displayName: 'Results',
     render: function () {
         return (
-            <div>Results</div>
+            <div className="results-section">
+                <TabbedArea defaultActiveKey={1}>
```
Comment (shidhincr) [inline app/scripts/ui/results.js]: :+1:

**Comment by shidhincr** [comment]
@ivinjose Looks good to me.. Merging this PR

**Pull request merged**